sass/_support.scss

```
@@ -64,6 +64,48 @@
         }
     }
 
+    #supporters {
+        display: flex;
+        flex-direction: column;
+        max-width: var(--page-max-width);
+        margin-inline: auto;
+
+        .supporters-section {
+            display: flex;
+            flex-direction: column;
+
+            h2 {
+                text-align: center;
+            }
+
+            .cards {
+                display: flex;
+                flex-direction: row;
+                flex-wrap: wrap;
+                gap: 1rem;
+
+                .supporters-card {
+                    display: flex;
+                    flex-direction: column;
+                    justify-content: center;
+                    align-items: center;
+                    border: 1px solid var(--borders-color);
+                    border-radius: 16px;
+                    padding-top: 4px;
+                    padding-bottom: 4px;
+                    padding-left: 8px;
+                    padding-right: 8px;
+
+                    img {
+                        aspect-ratio: 1/1;
+                        object-fit: contain;
+                        width: 150px;
```

```suggestion
                        width: var(--logo-width);
```

And defining `--logo-width` in the `#gold-supporters` and `#silver-supporters` sections as suggested below

---

Applied in c3118df625fdeb83ca1abdbbc26db8293f376db4